packages/frontend-acceptance-tests/src/features/3 - Negative Tests/1_adult_licence_12_NEG.feature

```diff
@@ -178,12 +178,23 @@ Scenario: Scenario 9 - Contact Details Errors
       | ErrorMessage  |
       | Choose how we should contact the licence holder  |
     And I click email radio button and click continue
-    And I am on the newsletter page and I click continue
+
+
+Scenario: Scenario 10 - Newsletter Errors
+# Scenario 10.1: Test Errors on Newsletter page -  Error messages - Nothing selected
```

Comment:
There are a lot of commented scenarios like this in this file - looks like Nabeel did it a few years ago. We should probably delete them as comments as I don't think they're useful anymore.